core.py

```diff
@@ -9,7 +9,10 @@ def __init__(self, bot: commands.Bot):
         self.bot = bot
         self.voiceBot = None
         self.audiofile = None
-        self.games = []
+        archive_po_allow = discord.Permissions(read_messages = True, read_message_history = True)
+        archive_po_deny = discord.Permissions.text()
+        archive_po_deny.update(read_messages = False, read_message_history = False, create_instant_invite = True)
```

Comment:
Constructor dont take dis?

Reply:
Easier to read this way. `discord.Permissions.text()` doesn't take arguments, and `discord.Permissions()` requires full list of parameters with `True` value or numeric value of permissions.